Sedimark/Sedimark-Toolbox
# feat: ngsild broker kubernetes manifests

## Description

Hi there! This PR adds kubernetes manifests to deploy an NGSI-LD context broker instance in a kubernetes cluster, based on the provided docker-compose in this repo, and largely inspired from [the one provided by Fiware](https://github.com/stellio-hub/stellio-k8s/tree/main) (although with some modifications). Please note that no ingress manifest is provided. This is because we do not intend to expose the broker API outside of the cluster. To access it anyway, you can port-forward (check the README for an example command to do so).

Contributes to #5 

## How to test

Should you want to test these manifests in your own cluster, just check the updated README to know all the environment variables you need to set, and what command to use to parse them in the manifests and deploy.

## To-do

- [x] do more testing of the API to validate the correct deployment of the app.
